src/ui/Filters/FilterOption.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Spring, animated } from 'react-spring';
+import Icon from 'ui/Icon/Icon';
+
+const Container = styled.div`
+  border-bottom: 0.5px solid ${props => props.theme.colors.terracota.main};
+`;
+
+const Header = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  height: 62px;
+`;
+
+const Checkbox = styled.div`
+  align-items: center;
+  background-color: ${props => (props.active ? props.theme.colors.bunting.main : 'transparent')};
+  border: 1px solid ${props => props.theme.colors.bunting.main};
+  display: flex;
+  height: 20px;
+  justify-content: center;
+  margin-right: 12px;
+  width: 20px;
+`;
+
+const OptionsContainer = styled(animated.div)`
+  overflow: hidden;
+  transition-timing-function: linear;
+`;
+
+const Option = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  margin-bottom: 24px;
+
+  @media ${props => props.theme.queries.desktop} {
+    &:hover ${Checkbox} {
+      background-color: ${props => props.theme.colors.bunting.main};
+    }
+  )};
+`;
+
+const Caret = styled.div`
+  margin-left: 16px;
+  margin-right: 10px;
+  transform: rotate(${props => (props.active ? '180deg' : '0deg')});
+  transition: transform 1s ease;
+`;
+
+const Counter = styled.span`
+  background-color: ${props => props.theme.colors.terracota.main};
+  border-radius: 50%;
+  box-sizing: border-box;
+  display: block;
+  height: 20px;
+  line-height: 20px;
+  margin: 0 10px;
+  margin-top: 1px;
+  padding-top: 1px;
+  text-align: center;
+  vertical-align: middle;
+  width: 20px;
+  color: ${props => props.theme.colors.white.main};
+  font-weight: 700;
+  font-size: ${12/16}rem;
+  font-family: ${props => props.theme.text.secondary};
+  text-align: center
+`;
+
+const SpringContainer = styled.div`
+  overflow-y: auto;
+  max-height: none;
+  
+  @media ${props => props.theme.queries.desktopLarge} {
+    max-height: 322px;
+  }
+`;
+
+const OptionName = styled.span`
+  color: ${props => props.theme.colors.bunting.main};
+  line-height: 18px;
+  margin-top 4px;
+  font-size: ${14/16}rem;
+`;
+
+const FilterName = styled.span`
+  color: ${props => props.theme.colors.terracota.main};
+  line-height: 14px;
+  margin-top 4px;
+  font-size: ${12/16}rem;
+  font-weight: 700;
+  font-family: ${props => props.theme.text.secondary};
+  text-transform: uppercase;
+  letter-spacing: 1.85px;
+  flex: 1
+`;
+
+class FilterOption extends React.Component {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    name: PropTypes.string.isRequired,
+    data: PropTypes.arrayOf(PropTypes.shape({})),
+    active: PropTypes.arrayOf(PropTypes.string),
+    onHeaderClick: PropTypes.func,
+    onChange: PropTypes.func,
+  };
+
+  static defaultProps = {
+    isOpen: false,
+    data: [],
+    active: [],
+    onHeaderClick: null,
+    onChange: null,
+  };
+
+  _handleChange = option => {
```

Comment:
Let's rename _all_ functions that start with these underscores. The underscores are unnecessary.